packages/react-dom/src/client/ReactDOM.js

```diff
@@ -71,6 +71,12 @@ import {
   enqueueStateRestore,
   restoreStateIfNeeded,
 } from '../events/ReactDOMControlledComponent';
+import type {ReactPriorityLevel} from 'react-reconciler/src/ReactInternalTypes';
+import {
+  getCurrentUpdateLanePriority,
+  schedulerPriorityToLanePriority,
+  setCurrentUpdateLanePriority,
```

Comment:
These should be imported from `ReactFiberReconciler` instead. Typically, renderers should never import methods from deep inside the `react-reconciler` package. Only from the entry point. The main exceptions are types or constants, but even those we should try not to leak to the renderers.